extension.json

```diff
@@ -48,14 +48,8 @@
 	"config": {
 	},
 
-	"Rights": {
-		"can-approve-pages": {
-			"description": "Allows users to approve pages",
-			"groups": []
-		}
-	},
 	"GroupPermissions": {
-		"sysop": {
+		"approvers": {
 			"can-approve-pages": true
 		}
 	},
```

Comment:
This still does not make sense and looks like it should be fully omitted

Reply:
Can we discuss how can we get Approvers then? Initially there are going to be Approvers with Zero categories"

ATM I am doing this.

['ug.ug_group' => self::APPROVERS_GROUP](https://github.com/ProfessionalWiki/PageApprovals/pull/17/files#diff-38cffc1abe7f3015793c75b879864b0516ec3665636a37b490de003dc4134358R33)

Reply:
Why would there be approvers with zero categories? Such users would not be able to approve anything anyway, since no categories will match.

Reply:
Of course, there are gonna be approvers with Zero Categories. Or are we planning to show thousands of users on this page?

![image](https://github.com/ProfessionalWiki/PageApprovals/assets/15813104/7beb740b-4f0c-4553-8e6c-1109ce75fd27)

Reply:
Oh, I see what you are talking about now. I was under the impression we were going to have an "Add user" button to add a new user row.

Reply:
Yes. Solving this wasn't really in the scope of this ticket but I gave it a shot anyway for testing since it was the easiest and fastest for me to implement. If not:

We can create an endpoint and respective functions to add a user_id to `approver_config` table. We can put "Add Approver" button on this page with a small input box. Or we can link this button to: 

`/index.php/Special:UserRights` and ask user to add it to the approvers group.


Perhaps it can be done in:

https://github.com/ProfessionalWiki/PageApprovals/issues/31

Reply:
What I wrote in the task:

> Table with user and categories columns. **Ability to add new rows**

The UI we are creating here is a management UI on top of the `approver_config` table. No need to complicate things by involving the rights system.

Reply:
By "Ability to add new rows" I understood, adding categories to the `approver_config` since `DatabaseApproverRepository` didn't have any method for adding `user` to it. Since everything else was there but not this method I assumed we were going to handle creating approvers some other way. Now here is what I am gonna do:

1- Create method for adding an approver in `DatabaseApproverRepository` and necessary functions.
2- Create a REST Endpoint to add an approver and put a button to add an approver on the UI